subgraphs/cryptopunk/src/mappings/cryptoPunkMarket.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  PunkOffered,
+  PunkBidEntered,
+  PunkBought,
+} from "../../generated/cryptopunkContract/cryptopunkContract";
+import { _User } from "../../generated/schema";
+import { getSellerAddressFromPunksOfferedForSale } from "../constants/utils";
+import { createBid } from "../constants/initializers";
+import { updateTrades } from "../modules/Trades";
+import * as constants from "../constants/constants";
+import { createUserCollectionAccount } from "../constants/initializers";
+export function handlePunkOffered(event: PunkOffered): void {
+  let sellerAddress = getSellerAddressFromPunksOfferedForSale(
+    event.params.punkIndex
+  );
+  createUserCollectionAccount(constants.TradeType.SELLER, sellerAddress);
```

Comment:
i'm also not sure if i would increase `sellerCount` for a listing that hasn't sold yet, but i do understand that it can be argued that it could since listing is on-chain (this is more ambiguous than the example for bid below)

i.e. most marketplaces are off-chain orderbook so listing an NFT for sale would not show up, meaning we would not count that as an increase in seller count